CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 * Optional tracking of previous data in changes. Set the `store_changed_from` trigger option.
 
+### Changed 
+
+* Added `@schema_prefix "carbonite_default"` on all schemas. This will enable the usage of `Repo.insert` etc on transactions without specifying a prefix, when using the default carbonite prefix.
```

Comment:
```suggestion
* Added `@schema_prefix "carbonite_default"` on all schemas. This will enable the manual usage of `Repo.insert/2` & friends on transactions without specifying a prefix, when using the default carbonite prefix.
```

:+1: